packages/ui/src/lib/NativeBase/Navbar/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { Image, Box, HamburgerIcon, HStack, Button, Square } from "native-base";
+import PropTypes, { InferProps } from "prop-types";
+import React, { useReducer } from "react";
+import MenuItem, { menuItemPropTypes } from "./menuItem";
+
+const navbarPropTypes = {
+  items: PropTypes.arrayOf(PropTypes.shape(menuItemPropTypes)).isRequired,
+  logo: PropTypes.shape({
```

Comment:
Perhaps call it `imageProps` for laziness and clarity or follow native base convention as `_image`.

Reply:
It's still logo

Reply:
I have removed the logo prop for Navbar In the latest commits because I think the image doesn't have to be modifiable from outside.

Reply:
I see